vendor-bin/propel2/composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "autoload-dev": {
+        "classmap": [
+            "../../tests/Bridge/Propel2/Model"
```

Comment:
But inherited from what? This namespace is manually mapped here because Propel doesn't allow PSR-4 IIRC.